user-guide/Getting_started/DaaS/Order_DataMiner_credits.md

```diff
@@ -0,0 +1,50 @@
+---
+uid: Order_DataMiner_credits
+---
+
+# Ordering DataMiner credits
+
+To create and maintain a DaaS instance, your [organization](xref:Pricing_Usage_based_service#organization) must have sufficient DataMiner credits. To order DataMiner credits for the first time, follow the procedure below.
+
+> [!NOTE]
+> If you have already created a SaaS subscription as detailed below, and you want to order **additional credits**, you do not have to create a new SaaS subscription. Instead, go to your SaaS subscription in the Azure portal, and click *Open SaaS Account on publisher's site* to be redirected to the *Purchase credits* page.
+
+1. Go to the [Azure Marketplace](https://azuremarketplace.microsoft.com/) and search for DataMiner.
+
+1. Click *Get It Now* to be redirected to the Azure portal.
+
+   > [!IMPORTANT]
+   > Make sure you have selected the plan *DataMiner credits*.
+
+1. Once you have been redirected to the Azure portal, click *Subscribe*.
+
+   This will start the process of creating a SaaS subscription resource to be deployed in a resource group. You will need to provide a resource name for the SaaS resource and assign the resource to a resource group of your choosing (or make a new resource group). Finally click on *Review + subscribe*.
```

Comment:
These are several steps in one, which something we prefer to avoid because it's not very user-friendly. Each numbered step should correspond to one separate action. The result of a step should be described in an indented line below the step itself when necessary. Could you adjust this so it's clear to the user what they should do, step by step?